materialize-snowflake/snowflake.go

```diff
@@ -528,9 +528,6 @@ func (d *transactor) Acknowledge(ctx context.Context) (*pf.ConnectorState, error
 	// the queries are actually executed and done
 	var results = make(map[string]stdsql.Result)
 	for stateKey, item := range d.cp {
-		if len(item.Query) == 0 {
```

Comment:
Since we are only ever fully deleting a key/value pair from `d.cp` or adding an item with a non-zero length query, I'm not adding this check to `materialize-databricks` and removing it from snowflake for consistency.